resource/js/hierarchy.js

```diff
@@ -128,76 +128,126 @@ function createConceptObject(conceptUri, conceptData) {
  * @param {Object} parentData 
  */
 function attachTopConceptsToSchemes(schemes, currentNode, parentData) {
```

Comment:
The changes in this file look more invasive than they are, because you've reformatted the code when editing it. Consider making it more like the original code so it's easier to see what has changed. However, I understand that this may be difficult to do afterwards.